core/src/main/scala/org/genivi/sota/core/CampaignResource.scala

```diff
@@ -74,7 +76,8 @@ class CampaignResource(db: Database, namespaceExtractor: Directive1[Namespace])
   }
 
   def extractId(ns: Namespace): Directive1[Campaign.Id] =
```

Comment:
It might be possible to use `WebService.allowExtractor`, something like this:

```
def extractId(ns: Namespace): Directive1[Campaign.Id] = 
  allowExtractor(namespaceExtractor, extractUuid, u => db.run(Campaigns.exists(Campaign.Id(u))).flatMap ... return Campaign.Id
```